docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -84,11 +98,27 @@ secret/medusa-bucket-key configured
 
 ### Create or update the k8ssandra-cluster
 
-Install the k8ssandra-cluster chart with the following properties. Backup and restore operations are enabled by default. In the following example, `bucketName` corresponds to the name of the S3 bucket: `K8ssanda-bucket-dev`.  The `bucketSecret` corresponds to the secret credentials.
+Install the `k8ssandra-cluster` chart with the following properties. You can reference the provided [backup-restore-values.yaml](./backup-restore-values.yaml) file. It contains:
+
+```
+size: 3
+backupRestore: 
+  medusa:
+    enabled: true
+    bucketName: k8ssandra-bucket-dev
+    bucketSecret: medusa-bucket-key
+    multiTenant: true
+    storage: s3
+```
+
+Example:
+
+`helm install k8ssandra-cluster-1 k8ssandra/k8ssandra-cluster -f backup-restore-values.yaml`
+
+Backup and restore operations are enabled by default. In the example YAML, `bucketName` corresponds to the name of the S3 bucket: `K8ssanda-bucket-dev`.  The `bucketSecret` corresponds to the secret credentials.
```

Comment:
> The `bucketSecret` corresponds to the secret credentials.

We should clarify that it relates to a Kubernetes Secret which contains the object store credentials. Bonus points if we illustrate what that secret should look like.

Reply:
Done, thanks